scripts/build.py

```diff
@@ -0,0 +1,79 @@
+import os
```

Comment:
I think we should move back this file to its original location for better tracking changes. It should be relocated next time.

Reply:
As the purposes of this PR, it tries to exclude all unneeded files from bdist wheels. the changes applied on this file after move is not much, you can compare them easily.